packages/sit-onyx/src/components/OnyxDataGrid/features/selection/types.ts

```diff
@@ -0,0 +1,44 @@
+import { type MaybeRef, type MaybeRefOrGetter } from "vue";
+import type { DataGridEntry } from "../../types";
+
+export type SelectionState = {
+  /**
+   * If the `contingent` set refers to entries that are included in the final selection or excluded from it.
+   * The `selectMode` is closely coupled with the `HeaderSelectionCheckbox` which is used to "select all" or "deselect all".
+   * It allows for tracking selections even when not all rows are known.
+   * This especially useful in case of pagination or asynchronous data loading.
+   *
+   * Based on this there are the following scenarios:
+   *
+   * | selectMode | selected.size |      result      | HeaderSelectionCheckbox |
+   * |------------|:-------------:|:----------------:|-------------------------|
+   * | "include"  |      == 0     | nothing selected | unchecked               |
+   * | "include"  |       > 0     |   some selected  | indeterminate           |
+   * | "exclude"  |       > 0     |   all selected   | checked                 |
```

Comment:
```suggestion
   * | "exclude"  |       == 0     |   all selected   | checked                 |
```

Shouldn't it be like this?